timeline/tests/test_templatetags.py

```diff
@@ -180,6 +180,21 @@ def test_get_app_icon_html_plugin(self):
         self.assertIn(plugin.title, ret)
         self.assertIn(url, ret)
 
+    def test_get_app_icon_html_example_site_app(self):
+        """Test get_app_icon_html() on event from an app plugin"""
+        plugin = get_app_plugin('example_site_app')
+        event = self._make_event(
+            project=self.project,
+            app='example_site_app',
+            user=self.user_owner,
+            event_name=TEST_EVENT_NAME,
+            description=TEST_EVENT_DESC,
+        )
+        ret = tags.get_app_icon_html(event, self.plugin_lookup)
+        self.assertIn(plugin.icon, ret)
+        self.assertIn(plugin.title, ret)
+        self.assertIn('', ret)
```

Comment:
This just asserts that an empty string is included in `ret`, probably returns true every time :)
I also recommend adding an assertion for the correct entry point URL being returned in the result, similar to what is done in ` test_get_app_icon_html_plugin()`.

Reply:
I have to say, that I am confused here( 
The thing is, that `reverse(entry_point, kwargs={'project':None})` raises exception => the url is being setted up to `''`. So in ret there will be an html without any `href=`. 
Maybe I misunderstood something((